### ISSUE TITLE
Resizing window on the top left coner is difficult on Windows

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#6333]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
Sometimes its  difficult to resize window from the top left corner. The other sides its easy to resize. Sometimes the resize pointer changes and instead of resizing the window it just moves the browser window. (Intermittent issue)

**Expected behavior:**
Should be able to resize window from the top left corner

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 x64

- Brave Version:
0.13.0 Preview 5

- Steps to reproduce:
    1. Bring the mouse cursor to the top left corner to change the cursor to double arrow changes
    2. Click the browser window to resize and cursor automatically changes back and instead of resizing the window it moves the window along
    3.

This is an intermittent issue and tricky to reproduce. 

- Screenshot if needed:

- Any related issues:
cc: @bsclifton

**Comment by bsclifton** [comment]
Duplicate of https://github.com/brave/browser-laptop/issues/4782; I'll make sure that issue mentioned the top left
